README.md

```diff
@@ -3,83 +3,23 @@
 
 # Report Official Development Assistance (RODA)
 
-This service enables the Department for Business, Energy and Industrial Strategy (BEIS) and their partner organisations to collect and report information on the spending of Official Development Assistance (ODA).
+This service enables the Department for Business, Energy and Industrial Strategy
+(BEIS) and their partner organisations to collect and report information on the
+spending of Official Development Assistance (ODA).
 
-## Getting started
-
-First, run the setup script. This installs the required system (assuming you're using OSX and Homebrew), frontend and Ruby dependencies, as well as setting up the test and development databases.
-
-```bash
-script/setup
-```
-
-Once setup has been completed, you can start the server with
-
-```bash
-script/server
-```
-
-and the tests should pass
-
-```bash
-script/test
-```
-
-## Running backing services with Docker compose
-
-If you prefer not to install the backing services (Postgres and Redis) with
-Homebrew via the scripts above, run them in the background with Docker and
-then use standard rails commands to interact with the application (you will need
-Docker installed on your device):
-
-```
-docker compose -f backing-services-docker-compose.yml up -d
-```
-
-To stop the backing services:
-
-```
-docker compose -f backing-services-docker-compose.yml down
-```
-
-## Architecture decision records
-
-We use ADRs to document architectural decisions that we make. They can be found in doc/architecture/decisions and contributed to with the [adr-tools](https://github.com/npryce/adr-tools).
-
-## Managing environment variables
-
-We use [Dotenv](https://github.com/bkeepers/dotenv) to manage our environment variables locally.
-
-The repository will include safe defaults for development in `/.env.example` and for test in `/.env.test`. We use 'example' instead of 'development' (from the Dotenv docs) to be consistent with current dxw conventions and to make it more explicit that the real values are not to be committed.
-
-To manage sensitive environment variables:
+In February 2023 BEIS became the Department for Science Innovation and
+Technology (DSIT), which had a large impact on the application. You will
+see references to both organisations in both the code and the data, and is worth
+keeping in mind as you work on the application.
 
```

Comment:
Should be "Department for Science, Innovation and Technology" and "...and this is worth keeping in mind"